TranchedLockupPresale/build/config.gypi

```diff
@@ -0,0 +1,67 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```

Comment:
we could skip ./build dir as far as I know

Reply:
Correct.